x-pack/plugins/observability/public/components/shared/exploratory_view/header/header.tsx

```diff
@@ -1,56 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import React from 'react';
-import { i18n } from '@kbn/i18n';
-import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
-import { TypedLensByValueInput } from '../../../../../../lens/public';
-import { useSeriesStorage } from '../hooks/use_series_storage';
-import { ExpViewActionMenu } from '../components/action_menu';
-import { useExpViewTimeRange } from '../hooks/use_time_range';
-import { LastUpdated } from './last_updated';
-import type { ChartTimeRange } from './last_updated';
-
-interface Props {
-  chartTimeRange?: ChartTimeRange;
-  lensAttributes: TypedLensByValueInput['attributes'] | null;
-}
-
-export function ExploratoryViewHeader({ lensAttributes, chartTimeRange }: Props) {
-  const { setLastRefresh } = useSeriesStorage();
-
-  const timeRange = useExpViewTimeRange();
-
-  return (
-    <>
-      <ExpViewActionMenu timeRange={timeRange} lensAttributes={lensAttributes} />
```

Comment:
This component is the one that renders the buttons in the kibana header:
<img width="477" alt="Screenshot 2022-02-03 at 15 13 54" src="https://user-images.githubusercontent.com/2249229/152359606-2ce6df67-8679-44b0-9cb7-db26bfe41801.png">
s

I don't see it moved anywhere else. Is that intentional?

Reply:
that was a mistake. let me think where it would make sense to put it